src/drl_graph.h

```diff
@@ -37,8 +37,14 @@
 #include "DensityGrid.h"
 #include "igraph_layout.h"
 
+#include <map>
+#include <vector>
+#include <ctime>
+
 namespace drl {
 
+using namespace std;
```

Comment:
I'd rather avoid `using namespace` in headers. I know that in this particular place it is in a `namespace {...}` block so the imported namespace should not leak out; however, there are still possibiities for [subtle bugs](https://stackoverflow.com/a/6175850/156771) that you can introduce this way.

Reply:
I fully agree with that, but how shall we fix things then? Add the `std::` in the headers where needed?

The `using namespace std;` was already all over the place in the DrL code.  I was trying to clean that up and move it into the namespace (at the same time as cleaning up the includes). What actually happened here was that `drl_graph.h` includes `DensityGrid.h` which has a naked `using namespace std;` (which I forgot to move into the namespace as well).

What I'm trying to say is that I didn't just add a `using namespace` arbitrarily here. I was actually trying to reduce the number of naked `using namespace`s.

What is your preference?

 1. add the `std::` in headers (not that much work)
 2. move the `using namespace std;` into the `drl` namespace everywhere
 3. something else? 

The DrL code is not the only one that has this in headers. The HRG code also does.

Reply:
We should probably add `std::` in the headers where applicable.

Reply:
I pushed a commit that takes care of this in more headers. I had to add `using namespace std;` to the cpp files sometimes.

I did not touch Bliss, I'll take up the issue with the author as he's preparing a new release anyway.

I did not touch simpleraytracer.

```
grep 'using namespace std' -R . --include '*.h*'

./optional/simpleraytracer/Point.h:using namespace std;
./optional/simpleraytracer/Light.h:using namespace std;
./optional/simpleraytracer/Shape.h:using namespace std;
./src/bliss/utils.hh:using namespace std;
```

Reply:
Okay, that's fine, thanks!